### PR TITLE
Release 97.0.0

### PR DESCRIPTION
**Pull request by yndajas**
* BREAKING: Remove ability to specify "temporary" redirects for Router API [PR](https://github.com/alphagov/gds-api-adapters/pull/1286)
  * Note: This is an unused feature, so should not be breaking in practice.
* Add publishing app to Host Content [PR](https://github.com/alphagov/gds-api-adapters/pull/1287)

---

This repo is owned by the publishing platform team. Please let us know in #govuk-publishing-platform when you raise any PRs.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,439 +1,449 @@
...
-# 71.2.0
+## 71.2.0
 
 * Add endpoints for authenticated "Save a page" API
 
-# 71.1.0
+## 71.1.0
 
-* Removed `LinkCheckerApi#upsert_resource_monitor` method as it’s not supported by Link Checker API itself.
+* Removed `LinkCheckerApi##upsert_resource_monitor` method as it’s not supported by Link Checker API itself.
```
Comment (brucebolt) [inline CHANGELOG.md:297]: Was this change intentional?  Similar question for release 61.0.0 and a few others.

Comment (yndajas) [reply CHANGELOG.md:297]: Oops

**Review by brucebolt — APPROVED** [review]
